internal/fields/model.go

```diff
@@ -64,4 +75,40 @@ func (orig *FieldDefinition) Update(fd FieldDefinition) {
 		}
 		orig.Fields = updatedFields
 	}
+
+	if len(fd.MultiFields) > 0 {
```

Comment:
Maybe extract the entire clause to a separate function?

Reply:
Done, we can consider using generics there when we update to 1.18 :) The updater for fields and for multi fields is basically the same.